io-engine/src/grpc/mod.rs

```diff
@@ -168,6 +173,45 @@ where
         .map_err(|_| Status::resource_exhausted("ENOMEM"))
 }
 
+/// Manage locks accross multiple grpc services.
+pub async fn acquire_subsystem_lock(
+    subsystem: &ResourceSubsystem,
+    resource: Option<String>,
+    global: bool,
```

Comment:
If not being used let's remove it to keep it simple. We can add later if required.

Reply:
fixed